prov/tcp/src/xnet.h

```diff
@@ -98,6 +98,9 @@ typedef void  xnet_profile_t;
 #define XNET_MIN_MULTI_RECV	16384
 #define XNET_PORT_MAX_RANGE	(USHRT_MAX)
 
+/* provider specific op flags */
+#define TCP_NO_CONNECT FI_TCP_NO_CONNECT
```

Comment:
Any reason not to use `FI_TCP_NO_CONNECT` directly?

Reply:
The provider specific flags are typically defined within the provider header files without and FI prefixing and used internally.  I wanted to have a definition here as a placeholder so that it's obvious there are existing provider op flags that need to be accounted for.

Reply:
@ooststep This is true when the API flag differs from the provider flag, or the scope of the flags are different.  E.g. the API flags cover a 64-bit range, but the provider flag maps to some 16-bit range in the protocol.